lib/assets/blog-article.mst

```diff
@@ -5,9 +5,9 @@
 </head>
 
 <body>
-<header class="extra-header">
```

Comment:
Yeah, missing the extra uses of the name was sloppy on my part, got all of them now.
And noted! thanks for the advice. But, in that commit, would it make sense to include the renaming in the CSS Files in that commit as well? Or just keep the CSS changes in one commit and the .mst file in the commit you're talking about?